arkham_lcg_scenarios_2021-02-08.sql

```diff
@@ -0,0 +1,79 @@
+# ************************************************************
+# Sequel Pro SQL dump
+# Version 4541
+#
+# http://www.sequelpro.com/
+# https://github.com/sequelpro/sequelpro
+#
+# Host: 127.0.0.1 (MySQL 5.7.33)
+# Database: arkham_lcg_scenarios
+# Generation Time: 2021-02-08 13:12:50 +0000
+# ************************************************************
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table scenarios
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `scenarios`;
+
+CREATE TABLE `scenarios` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(50) DEFAULT NULL,
+  `cycle` varchar(50) DEFAULT NULL,
+  `position` tinyint(2) DEFAULT NULL,
+  `standalone` tinyint(1) DEFAULT NULL,
+  `owned` tinyint(1) DEFAULT NULL,
```

Comment:
- for the binary fields you could use 'unsigned' checkbox in table structure tab to ensure no minus figures can be entered. I guess you won't allow user to input minus as you'll prob use a checkbox but thought so it's likely not an issue

Reply:
I think tiny int is the most stable way of doing boolean in SQL as check box isn't always supported